docs/topics/development/testing.md

```diff
@@ -0,0 +1,134 @@
+# Testing
```

Comment:
I don't think we need this in this PR.

Reply:
Wdym? I used this process _in_ this PR.

Reply:
Testing docs are not related to the issue you're fixing. Please file a new issue and attach the changes to a new PR fixing that new issue.

Reply:
I don't think that's the right approach. We've added a check in the PR template specifically requesting that [documentation be updated](https://github.com/mozilla/addons-server/blob/master/.github/PULL_REQUEST_TEMPLATE.md) along with the PR where it is relevant to change.

This PR required making changes to `.xpi` files to make tests pass, that not only makes the change "related to the issue you're fixing" but serves as a good opportunity to document something that wasn't previously documented.

Doing that in a separate issue/PR is:
- more work for no benefit
- risks losing track of why the docs were updated in the first place

Reply:
If we have to revert this patch, aren't the changes to the docs in this PR still relevant?  That's why it should be its own PR.

Reply:
If we have to revert the patch, we can still leave the docs. I don't think the overhead is worth the limited safety in this and in most cases.

Reply:
Also, who knows maybe if we have to revert the patch the docs are also invalid... it's a total hypothetical so I can't base decisions off of it.

Reply:
The format change makes it difficult to identify the relevant changes. I would agree it's related to this PR if it was just about documenting the `zip` stuff but with the format change it's a bit annoying to follow... Ideally format change should happen in a separate PR and this one should contain only your additions.

Reply:
Okay, happy to compromise there. For context, I converted the file to markdown in spirit of https://github.com/mozilla/addons/issues/9389

One way to migrate our documentation to markdown is to convert as we go. I can see how it would make it harder to diff though.